components/Instructions/editRecipeInstructions.jsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+
+function EditRecipeInstructions({ recipes }) {
+  const [instructions, setInstructions] = useState([]);
+
+  useEffect(() => {
+    if (recipes) {
+      const fetchRecipeInstructions = async () => {
+        const response = await fetch(`/api/recipes/${recipe.id}/instructions`);
```

Comment:
This can be moved to the db helpers.

Reply:
@Mpho-vincent-makgwe please add this as helper function

Reply:
I understand.
We'll take a look at implementing best practices later.